src/components/ha-selector/ha-selector-number.ts

```diff
@@ -81,17 +81,11 @@ export class HaNumberSelector extends LitElement {
     `;
   }
 
-  private get _value() {
-    return this.value ?? (this.selector.number?.min || 0);
-  }
-
   private _handleInputChange(ev) {
     ev.stopPropagation();
     const value =
       ev.target.value === "" || isNaN(ev.target.value)
-        ? this.required
-          ? this.selector.number?.min || 0
-          : undefined
```

Comment:
@bramkragten Did you known why we have this ?

Reply:
If the consumer of the selector handles required correctly I think I agree...